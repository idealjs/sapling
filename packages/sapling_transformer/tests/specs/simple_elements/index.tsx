const template = <h1>template</h1>;
