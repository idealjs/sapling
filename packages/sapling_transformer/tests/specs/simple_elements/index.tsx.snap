---
source: packages/sapling_transformer/tests/specs/simple_elements/index.tsx
expression: index.tsx
---
# Input
```tsx
const template = <h1>template</h1>;

```

# Transformations
```tsx
const template = (() => {
  var _el$ = _$createElement("h1");
  _$insertNode(_el$, _$createTextNode(`template`));
  return _el$;
})();

```
